### ISSUE TITLE
wB97x s8 parameter is negative

### ISSUE THREAD

**Issue by atom-moyer** [dftd4/dftd4#16]
Hi all,

I am using the D4 dispersion correction with a wB97x functional, however, I have come across the fact that the s8 parameter is negative. There are safe checks against negative values of s8 in the code. 

Is it safe to use the wB97x parameters?

Thanks,
Adam

**Comment by awvwgk** [comment]
Yes, for some functionals the `s8` is slightly negative. Usually a negative `s8` is an unphysical choice but we could not avoid it for every functional in the fit, we checked the effects of the negative `s8` for those functionals and it turns out they are fine.

I should check again, if I introduced a safe check against negative parameters that is blocking the usage of these parameters.

**Comment by atom-moyer** [comment]
Great! I will be doing some calculations with it, so I can let you know if does obviously wild things.

**Comment by reinhacr** [comment]
Hi there, I was looking into historical posts trying to find the source of the negative s8 parameter and I guess I tracked it down! I replaced the wb97x s8 and other parameters with the newer parameters from the wb97x-3c paper as mentioned in the SI of the HS13L paper. Using the wb97x parameters as is with the negative s8 parameter gave much lower dispersion energies than other functionals, as well as the newer parameters. For context, I have a large supramolecular complex (280 atoms, -12 charge) which when a cationic guest is bound inside, is -11 charge and slightly more atoms. The binding is non covalent in nature and hence why I've been reading alot of benching marking papers like HS13L trying to establish the suitability of those workflows to these complexes. Anyway, I do believe that wb97x-D4 parameters are leading to wild behavior. When comparing Edisp for the complex mentioned above to the value of E disp for other functionals and the corrected terms in wb97x-3c paper range from ~ -0.6 Eh (PBE) to -1.0 Eh (more if you use HF). The value with old parameters for wb97x is **-0.15 Eh** , where as with the new parameters it is **-0.94 Eh** which is a huge change, and the new parameters give a much more reasonable value in comparison to the other functionals and values I've seen for these complexes. -0.15 hartrees is very low in comparison to what I've seen spit out in some electronic structure codes when working with these systems (granted these were D3 corrections I was computing). 

Thank you! Feel free to follow up with questions or if you need notes on my install or complex. I hope that made sense- I'm a biochemist by training working with some larger supramolecular complexes in my postdoc so I admit I'm still trying to get these things down myself.

**Comment by reinhacr** [comment]
Hi, I did some more tests as I wanted to be sure it wasn't just my system, as its large and charged as discussed above, so I built a simple benzene dimer. The following is an excerpt of my spreadsheet. To be clear, I did not optimize the benzene dimer with DFT first or anything, just built in Avogadro, did a quick geometry clean (UFF), which resulted in a visually good geometry of what I wanted, and proceeded. I then used the standalone dftd4 package for the d4 entries, and the d3 entries were with the standalone s-dftd3 package. I copied the full values from the program into the spreadsheet, but have the full entries displayed in number as opposed to scientific mode, but values of 0.00 are not such. I did not pass any other options besides explicitly declaring the charge to be zero for the dftd4 and specifying the functional, besides in the case of the HS13L reading in the new parameters with the addition of the "--param 1.0, 0.4485, 0.3306, 4.2790" flag to the command. The output I verified that these were read. Granted, the functionals are different from each other as expected but all other d3 to d4 differences are relatively small and consistent, besides the wb97x which is 2 kcal/mol different. I could have posted a new issue, but I figure I'd leave it here and just tag the relevant parties. @awvwgk 



BENZENE DIMER | MONOMER 1 | MONOMER 2 | DIMER | dE_disp_binding (dimer-(monomer+monomer) converted to kcal/mol
-- | -- | -- | -- | --
wb97x zero dampening-d3 | 0.00 | 0.00 | -0.01 | -3.48 |  
wb97x d3 | -0.02 | -0.02 | -0.05 | -3.74 |  
b3lyp d3 | -0.02 | -0.02 | -0.05 | -5.32 |  
b3lyp d4 | -0.02 | -0.02 | -0.04 | -5.57 |  
wb97xd d4 negative s8 default params | 0.00 | 0.00 | 0.00 | -1.21 |  
wb97xd d4 params published in HS13L SI and wb97x-3c paper | -0.02 | -0.02 | -0.04 | -3.65 |
